README.md

```diff
@@ -68,7 +68,7 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 [rl-debian8]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.2/powershell_6.0.2-1.debian.8_amd64.deb
 [rl-debian9]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.2/powershell_6.0.2-1.debian.9_amd64.deb
 [rl-centos]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.2/powershell-6.0.2-1.rhel.7.x86_64.rpm
-[rl-ai]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.2/PowerShell-6.0.2-x86_64.AppImage
+[rl-ai]: https://github.com/PowerShell/PowerShell/releases/download/v6.0.1/PowerShell-6.0.1-x86_64.AppImage
```

Comment:
Was this intentional? Isn't related to preview, wasn't sure....

Reply:
Fixed a broken link. We stopped building AppImage since 6.0.1

Reply:
Filed https://github.com/PowerShell/PowerShell/issues/6483